docker/environment/blueprint.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: xl/v1
+kind: Blueprint
+metadata:
+  projectName: Create infrastructure and environment for Docker
+  description: |
+    This blueprint defines an environment to which Docker containers can be deployed
+  author: XebiaLabs
+  version: 1.0
```

Comment:
You can now add instructions if you like :)

Reply:
I've added tests, instructions and `USAGE-*.md` files.

I did not add Docker Compose files because this blueprint is about running in Docker. Using Docker Compose to spin up Docker seems kind of, erm, impossible. ;-)